lib/material_input/base_material_input.dart

```diff
@@ -170,6 +177,10 @@ class BaseMaterialInput extends FocusableMixin
     }
   }
 
+  /// Displays character count even if maxCount is not defined.
```

Comment:
The explanation of this field given in a later file is better (in particular because there is no notion of "not defined" in Dart, members always have a value):

> Whether or not the character count will be displayed when maxCount is null.